code/_onclick/hud/_defines.dm

```diff
@@ -154,6 +154,12 @@
 #define ui_bot_radio "EAST-1:28,SOUTH:7"
 #define ui_bot_pull "EAST-2:26,SOUTH:7"
 
+//Ghosts
+#define ui_ghost_jumptomob "SOUTH:6,CENTER-2:24"
+#define ui_ghost_orbit "SOUTH:6,CENTER-1:24"
+#define ui_ghost_reenter_corpse "SOUTH:6,CENTER:24"
+#define ui_ghost_teleport "SOUTH:6,CENTER+1:24"
+
```

Comment:
eventually we should do the pAI thing too.

Reply:
Absolutely agree, but that would mean tinkering with how pAIs are chosen, so I decided to move to the future PR(s).

Reply:
Yeah, that's fine. This is more important and allows @AffectedArc07 to move forward with Parallax.